src/js/chapterEnd.js

```diff
@@ -0,0 +1,39 @@
+//import renderContent function. 
+
+//quiz will be a node that can be embedded into chapter end page. 
+const createChapterEnd = (chapterNumber, quiz, nextPage, homePage) => {
+    const base = document.createElement("div");
+    base.setAttribute('id', 'chapter-end')
+
+    let quizPassed = false;
+    let toggleQuizStatus = () => quizPassed = true; 
+
+    let text = document.createElement("p");
+    text.innerHTML = `Chapter ${chapterNumber} complete!`
```

Comment:
Maybe we should avoid using innerHTML just to insert text into node and use textContent instead? InnerHTML have some problems, more details here -> https://www.geeksforgeeks.org/what-is-the-disadvantage-of-using-innerhtml-in-javascript/

Reply:
Yeah textContent seems better to do that.

Reply:
Thanks for sharing that @mdlugosz1 , learned somethin new =)